croud/__main__.py

```diff
@@ -468,6 +469,25 @@
                 "help": "List all organizations the current user has access to.",
                 "resolver": organizations_list,
             },
+            "edit": {
+                "help": "Edit the specified organization.",
+                "extra_args": [
+                    Argument(
+                        "--name", type=str, required=False,
+                        help="The new organization name to use.",
+                    ),
+                    Argument(
+                        "--plan-type", type=int, required=False,
+                        choices=[1, 2, 3, 4, 5, 6],
+                        help="The new support plan to use for the organization.",
```

Comment:
Should we add the limitations to the help text?

```suggestion
                        help="The new support plan to use for the organization. Argument is for superusers only.",
```

/cc @lukas-bernert

Reply:
I'd be in favor of doing it. However, we already have commands that are superuser only (e.g. croud monitoring..). If we want to implement it here we should be consistent and also think about adding it to other superuser-only commands. Either now or in the future.

Reply:
I think there is a difference between superuser commands, and superuser arguments. Afaik, `--plan-type` is the only argument that is only relevant for superusers. Everywhere else, the whole command is superuser only.
I don't mind, not adding the limitation to the help text for now.

Reply:
True. I'd add it for now. It helps for a better UX IMO.